### PR TITLE
Fix backslashes in graphviz causing incorrect rendering on windows.

### PR DESCRIPTION
**Pull request by Sushisource**
Simple fix to replace backslashes in node paths with forward slashes so the generated graphs render properly on windows instead of having tabs and newlines and whatnot all over the place.

### CODE REVIEW

File: src/graphviz.cc [hunk 1]
```diff
@@ -22,7 +23,9 @@ void GraphViz::AddTarget(Node* node) {
   if (visited_nodes_.find(node) != visited_nodes_.end())
     return;
 
-  printf("\"%p\" [label=\"%s\"]\n", node, node->path().c_str());
+  std::string pathstr = node->path();
```
Comment (nico) [inline src/graphviz.cc]: Can you remove the std::s? ninja has lots of `using namespace std`s.

**Comment by nico** [comment]
Thanks for the patch!

**Comment by Sushisource** [comment]
Removed the stds (heh, stds).

**Comment by nico** [comment]
Thanks for the patch!

It's strange that this is needed; the graphviz docs claim """In quoted strings in DOT, the only escaped character is double-quote ("). That is, in quoted strings, the dyad \" is converted to "; all other characters are left unchanged. In particular, \\ remains \\. Layout engines may apply additional escape sequences."""

I suppose the layout engine screws things up?

**Comment by Sushisource** [comment]
Yeah, almost certainly just a case of failing to conform to the spec.

**Pull request merged**